src/ripple/app/misc/impl/TxQ.cpp

```diff
@@ -1116,12 +1093,12 @@ TxQ::processClosedLedger(Application& app,
         return;
     }
 
+    std::lock_guard<std::mutex> lock(mutex_);
```

Comment:
Here's the crux of the change, so `feeMetrics_.update()` happens under the lock?